malaria24/ona/tests/test_tasks.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+
+import pkg_resources
+import pytest
+import responses
+
+from malaria24.ona.models import ReportedCase
+from malaria24.ona.tasks import ona_fetch_reported_cases
+
+
+@pytest.mark.django_db
+class OnaTest(TestCase):
+
+    def setUp(self):
+        responses.add(responses.GET, 'https://ona.io/api/v1/',
+                      status=200, content_type='application/json',
+                      body=pkg_resources.resource_string(
+                          'malaria24', 'ona/fixtures/responses/catalog.json'))
+        responses.add(responses.GET, 'https://ona.io/api/v1/data/79925',
+                      status=200, content_type='application/json',
+                      body=pkg_resources.resource_string(
+                          'malaria24', 'ona/fixtures/responses/data.json'))
```

Comment:
@bruskiza this is how we fake API call responses for Ona. When the HTTP API request is made the data from the JSON file on disk is returned instead of an actual API call happening to Ona during the test run.